Rozwiązanie_zadania_2

```diff
@@ -0,0 +1,50 @@
+def procedura1():
+   lista1 = [1, '2', '-5', 4 * 4, 12]
+   lista2 = [1, 1, 1, 1, 1]
+   lista1 = [int(i) for i in lista1]
+
+   print("\n" "Wymagany warunek: wartość z listy1> wartośći z listy2")
+
+   for i in range(len(lista2)):
+      a = lista1[i]
+      b = lista2[i]
+      if b > a:
+         print("\n" "Wartość %s z listy1 spełnia wymagany warunek" % b)
+      else:
+         print("\n" "Ta pozycja z listy1 nie spełnia wymaganego warunku")
+
+
+def procedura2():
+   slownik1 = {"a": 1, "b": 2, "c": 3}
+   slownik2 = {"a": 1, "c": 3, "d": 4}
+   lista1 = ['a', 'b', 'x', 'd']
+   print("slownik1=", slownik1)
+   print("slownik1=", slownik2)
+   print("lista1", lista1)
+   print("\n")
+   for i in lista1:
+      a = (slownik1.get(i, 0))
+      b = (slownik2.get(i, 0))
+      print("Wartość dla klucza %s: %s+%s=%s" % (i, a, b, a + b))
+
+def procedura3():
+   slownik1 = {"a": 1, "b": 2, "c": 3}
+   slownik2 = {"a": 1, "c": 3, "d": 4}
+   lista1 = ['a', 'b', 'x', 'd']
+
+   for i in range(len(lista1)):
+      a = ([(i, "%s" % slownik1.get(i, 0)) for i in lista1])
+      b = ([(i, "%s" % slownik2.get(i, 0)) for i in lista1])
+      if a >=b:
```

Comment:
tutaj trzeba stworzyc nowy slownik, a dopiero potem go wyswietlic.

Reply:
przed forem mozesz zdefiniowac sobie nowy slownik, np 
slownik3 = {}
for - tutaj mozna zamiast iterowac sie po dlugosci lista1 mozna zrobic tak:
for key in lista1:
 ...
wtedy pod key bedzie wpadac kolejny element z lista1 do uzycia jako klucz przy tworzeniu nowego slownika.
w tym ifie ponizej "if a >=b:"
mozesz stworzyc nowy element nowego slownika:
slownik3[key] = a
else: 
slownik3[key] = b

potem print(slownik3) zalatwi wymaganie koncowe zadania.